### PR TITLE
Chore/documentation

### PR DESCRIPTION
**Pull request by fredrikved**

### CODE REVIEW

**Comment by andreasgan** [comment]
@fredrikved I changed to `docker compose -d --wait" to wait for healthy. didnt help much because it wasnt truly ready immediately but yea, still nice to show that we tried lol.

But can you set up the project from scratch once to complete this fully? I still get stuck. I ran `make install` but it prompts for a db password for "btv".

**Review by andreasgan — CHANGES_REQUESTED** [review]
see comment

**Review by andreasgan — DISMISSED** [review]
I got it up and running now.
Haven't verified if the process is smooth fully from scratch but I think we're close.

File: cms/Makefile [hunk 1]
```diff
@@ -11,15 +11,14 @@ build:
 
 s:
 	make build
-	docker-compose up -d
+	docker compose up -d --wait
```
Comment (KillerX) [inline cms/Makefile:14]: image
...

**Review by KillerX — APPROVED** [review]
I think we can just merge this and then work more in small iterations

**Review by KillerX — APPROVED** [review]

**Pull request merged**